Add Meinberg Lantime M300

zeridon opened the pull request:
Adding Meinberg Lantime M300 hardware ntp clock (including Elevation images)

ravench commented on elevation-images/Meinberg/meinberg-lantime-m300.back.png:
Please crop these images to only include the part that is visible when the device is rack mounted, as these are used in the rack display.

ravench commented on device-types/Meinberg/lantime-m300.yml:
```diff
@@ -0,0 +1,32 @@
+manufacturer: Meinberg
+model: Lantime M300
+slug: lantime-m300
```
```yaml
slug: meinberg-lantime-m300

ravench commented on device-types/Meinberg/lantime-m300.yml:
```diff
@@ -0,0 +1,32 @@
+power-ports:
+  - name: PWR1
+    type: iec-60320-c14
+    label: PWR1
```
```yaml
maximum_draw: 20
```

ravench commented:
> I've requested photos for that speciffic variant from meinberg. Will update when available.
What about `https://www.adeptor.no/Media/Cache/Images/3/1/WEB_Image_Meinberg_LANTIME_M300_GPS_AD10__19__rack_lantime_m300_rear1621739275_plid_33291.jpeg` ?

zeridon commented:
> > I've requested photos for that speciffic variant from meinberg. Will update when available.
> 
> What about https://www.adeptor.no/Media/Cache/Images/3/1/WEB_Image_Meinberg_LANTIME_M300_GPS_AD10__19__rack_lantime_m300_rear1621739275_plid_33291.jpeg ?

I am ok with it. Will just remove the second PSU (as the image is with just 1)

zeridon commented:
Changed the elevation image and removed the Second PSU.

To quote mainberg (from a ticket request):
> Unfortunately, we are unable to take any photos as we only build customised devices just with an order. You can find graphics of the rear panel in the system manual.

E.g. the devices are very flexible (to build) but hard to generalize. This seems the better one. PSU's and others can be added easily
